examples/blob2d-outerloop/CMakeLists.txt

```diff
@@ -6,9 +6,7 @@ if (NOT TARGET bout++::bout++)
   find_package(bout++ REQUIRED)
 endif()
 
-bout_add_example(blob2d-outerloop
-  SOURCES blob2d.cxx
-  EXTRA_FILES data/BOUT.inp)
+bout_add_example(blob2d-outerloop SOURCES blob2d.cxx)
```

Comment:
data/BOUT.inp not needed? Thought it should be there so it's copied into the build directory

Reply:
`bout_add_example` copies it by default, along with `README.md` if that exists :)